### PR TITLE
add paypal service and default payment method

### PR DESCRIPTION
**Pull request by dara123**

### CODE REVIEW

File: src/Entities/PayPalAccount.php [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+    {
+        return $this->getLink('approval_url');
+    }
+
+    /**
+     * @return null|Customer
+     */
+    public function getCustomer()
+    {
+        if ($this->hasEmbeddedResource('customer')) {
```
Comment (XaosSintez) [inline src/Entities/PayPalAccount.php]: I think you can put ternary operator here and below

**Comment by XaosSintez** [comment]
Please add Entity and Service tests in `ApiTest` class

**Comment by XaosSintez** [comment]
Everything else seems ok

**Comment by XaosSintez** [comment]
:+1:

File: src/Entities/Schema.php [hunk 2]
```diff
@@ -102,6 +102,18 @@ public function __construct()
             'payment-cards/{cardId}' => function (array $content) {
                 return new PaymentCard($content);
             },
+            'paypal-accounts' => function (array $content) {
+                return new Collection(new PayPalAccount(), $content);
+            },
+            'paypal-accounts/{paypalAccountId}' => function (array $content) {
+                return new PayPalAccount($content);
+            },
+            'paypal-accounts/{paypalAccountId}/activation' => function (array $content) {
```
Comment (slavcodev) [inline src/Entities/Schema.php]: I believe this builder and next (`paypal-accounts/{paypalAccountId}/deactivation`) does not make sense. And seems never used.

**Pull request merged**